src/main/java/com/entropyteam/entropay/employees/envers/UserRevisionListener.java

```diff
@@ -0,0 +1,13 @@
+package com.entropyteam.entropay.employees.envers;
+
+import com.entropyteam.entropay.auth.AuthUtils;
+import org.hibernate.envers.RevisionListener;
+
+public class UserRevisionListener implements RevisionListener {
+
+    @Override
+    public void newRevision(Object revisionEntity) {
+        var auditEnversInfo = (AuditEnversInfo) revisionEntity;
+        auditEnversInfo.setUsername(AuthUtils.getUsername());
```

Comment:
Personally, I do not like the "Utils" classes. You can [search ](https://www.google.com/search?q=java+util+class+with+static+methods+antipattern&oq=java+util+class+with+static+methods+anti&gs_lcrp=EgZjaHJvbWUqBggCECEYFTIGCAAQRRg5MgcIARAhGKABMgYIAhAhGBXSAQkxMjYyM2owajeoAgCwAgA&sourceid=chrome&ie=UTF-8) about

I think that you can use the `SecurityContextHolder` directly in this method